ocs_ci/utility/azure_utils.py

```diff
@@ -220,6 +220,24 @@ def get_vm_instance(self, vm_name):
         )
         return vm
 
+    def get_vm_power_status(self, vm_name):
+        """
+        Get the running status of VM
```

Comment:
shouldn't it be `Get the power status of VM`?

Reply:
yes. Updated